### PR TITLE
US93253, Fixed IWD One Page Checkout latest version which conflict ou…

### PR DESCRIPTION
**Pull request by jeevitha90**
US93253, Fixed IWD One Page Checkout latest version which conflict our Heartand Magento Extension

### CODE REVIEW

**Comment by slogsdon** [comment]
@jeevitha90 When running locally after clearing Magento and browser caches, I'm seeing this:

screen shot 2017-10-04 at 10 30 47 am

which seems to be caused by our main /js/securesubmit/checkout-form.js file not loading:

screen shot 2017-10-04 at 10 34 06 am

Is this working on your end?

**Comment by slogsdon** [comment]
@jeevitha90 The assets are showing now, and the card tokenizes successfully. After tokenization, however, the checkout sits without proceeding to the order being created:

screen shot 2017-10-04 at 1 29 08 pm

I'm also going to loop in the UX/UI team to address the layout issues with the fields being confined in IWD's interface.

Are you able to test PayPal and MasterPass transactions as well to ensure their functionality is present? Without attempting payment with PayPal, I'm seeing this error appear:

screen shot 2017-10-04 at 1 28 55 pm

If you think it will be too much work to get PayPal and MasterPass working, just prevent the PayPal JS error from occurring, and we'll get additional stories for supporting IWD and PayPal/MasterPass.

**Comment by jeevitha90** [comment]
Hi Shane,

I have prevented the PayPal JS error, when without loading HPS PayPal. 
Can you able to create additional User Story to check PayPal and MasterPass is working in IWD extension.

**Comment by slogsdon** [comment]
@jeevitha90 I'll get the stories created.

The checkout is still halting after the single-use token is obtained:

screen shot 2017-10-05 at 9 19 29 am

Have you been able to figure out the issue there?

**Comment by slogsdon** [comment]
@jeevitha90 I'm seeing an issue when there's a tokenization error. The checkout keeps the overlay on top of the form, so the consumer cannot proceed without doing a hard refresh of the page. Are you able to look into this?

**Comment by jeevitha90** [comment]
@makenziemoore - I'm seeing some CSS issue

1. When mouse over the label it seems to be jump.
label_jumps

2. When I disable IWD extension, design gets changes
css_issue

File: app/design/frontend/base/default/template/securesubmit/storedcards.phtml [hunk 1]
```diff
@@ -9,8 +9,9 @@ $_storedCards = Mage::helper('hps_securesubmit')->getStoredCards(Mage::getSingle
             <h2><?php echo $this->__('Saved Credit Cards') ?></h2>
 
         <ul id="hps-saved-cards" class="saved-card-table">
-            <?php foreach ($_storedCards->getData() as $card): ?>
+            
                 <li>
+                  <?php foreach ($_storedCards->getData() as $card): ?>
```
Comment (slogsdon) [inline app/design/frontend/base/default/template/securesubmit/storedcards.phtml]: @makenziemoore Moving this line here would create broken HTML (one opening `li`, bunch of closing `li`).

Comment (slogsdon) [reply app/design/frontend/base/default/template/securesubmit/storedcards.phtml]: not a show stopper, so pushing this out to next deployment

**Pull request merged**